gentoo/gentoo
# net-im/synapse: add 1.82.0, drop 1.77.0, 1.78.0, add new dep pkg dev-python/immutabledict

- synapse version bump to 1.82.0
   - revised initd script, use `command_user="synapse:synapse"` rather than unnecessary `start_stop_daemon_args`
   - add `dev-python/pyicu` to test bdeps
   - change `dev-python/frozendict` to `dev-python/immutabledict`
- add new python package `dev-python/immutabledict`.
   - The package will be needed in next synapse version. It is a replacement for `dev-python/frozendict`, see https://github.com/matrix-org/synapse/pull/15113.
   - Tests pass for all 3 python versions.
- drop 1.77.0, 1.78.0
